stream-chat-android-state/src/main/java/io/getstream/chat/android/offline/utils/internal/ChannelMarkReadHelper.kt

```diff
@@ -86,7 +86,7 @@ internal class ChannelMarkReadHelper(
                     return false
                 }
 
-                if (!clientState.isOnline) {
+                if (!clientState.isNetworkAvailable) {
                     logger.i { "Cannot mark read because user is offline!" }
```

Comment:
What do you think about updating the logger's message too, such as "Cannot mark read because the internet is not available"?

Reply:
I'm going to change it slightly to 

"Cannot mark as read because network connection is not available"

Feel free to voice your opinion if you feel this is wrong. :D

Reply:
Looks perfect 💯